PhysiCell/custom_modules/custom.cpp

```diff
@@ -207,19 +208,28 @@ void custom_function( Cell* pCell, Phenotype& phenotype , double dt )
 void contact_function( Cell* pMe, Phenotype& phenoMe , Cell* pOther, Phenotype& phenoOther , double dt )
 { return; }
 
+
+
+//letters not in the human amino acid alphabet: b,j,o,u,x,z
+static const double alphabet[] {'a','c','d','e','f','g','h','i','k','l','m','n','p','q','r','s','t','v','w','y'};  // humman amino acide alphabet
+static const double pad {0};
+
+// number of matching antigen antibody amino sequences that account for 100% affinity.
+static double aminoComplete = 10;
```

Comment:
This seems like it could be the length of the shorter molecule between the antigen or antibody, which is 11. We can always change these things as we read more papers.

Reply:
I just set something for testing that seems reams reasonable. 
feel free to do other settings.